media/css/firefox/family/components/_hero.scss

```diff
@@ -10,6 +10,11 @@
 .c-hero {
     background: f3.$violet-dark center / cover no-repeat url('/media/img/firefox/family/grid-hero.svg');
 
+    .mzp-l-content.mzp-t-content-xl {
+        // not sure if this is the best place to set this max-width
+        max-width: f3.$section-max-width;
```

Comment:
just wanted to point this out - not sure if you already had an idea of where to set this, but didnt want to create conflicts if you were working in shared files

Reply:
Ah, good point, I forgot to update this in the issue. I made a [change to the section content containers](https://github.com/mozilla/bedrock/commit/d4e91ac09b3b8ebd750630159f86281e114b25fd) to use the Protocol xl width, which is 1152px (close enough to 1250px) instead of the default max-width on `mzp-l-content` (1440px).

If we want to stick with the 1250px width for content containers, let's add this style rule to the main `styles` partial with a comment that it's a protocol override. 

```
// F3 xl width override
.mzp-l-content.mzp-t-content-xl {
    max-width: 1250px
}
```